drivers/ieee802154/ieee802154_dw1000.c

```diff
@@ -1584,6 +1548,15 @@ static int dw1000_init(struct device *dev)
 
 	LOG_INF("GPIO and SPI configured");
 
+	/* Copy the phy configuration table */
+	const struct dwt_phy_config *rf_cfg = dwt_phy_cfg_get(ctx->phy_cfg_id);
```

Comment:
Please declare the variables in the beginning of the block. Also remove the empty line between variable setting and the if() statement.